wca-regulations.md

```diff
@@ -461,6 +461,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - C1b) During the solve, the competitor must use only one hand to operate the puzzle. Penalty: disqualification of the attempt (DNF).
         - C1b2) If a puzzle defect occurs, and the competitor chooses to repair it, they must repair it using only the solving hand. Penalty: disqualification of the attempt (DNF).
         - C1b3) If other body parts of the competitor come in contact with the puzzle or parts of the puzzle without the competitor's intention and without applying moves, this is not considered operating the puzzle, at the discretion of the judge.
+        - C1b4) During the solve, the competitors may choose to put the puzzle on the surface.
```

Comment:
How about this?

```suggestion
        - C1b4) During the solve, the competitor may hold the puzzle against the surface to help operate the puzzle.
```

Alternatively, I think it would be fine to reuse the wording from C1b+++ .

Reply:
This is much better!  I will remove C1b+++ and move this one to Chapter A and make sure it applies to all speedsolving.